app/src/main/java/com/rai/calculator/CalculatorFragment.kt

```diff
@@ -0,0 +1,74 @@
+package com.rai.calculator
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.rai.calculator.databinding.CalculatorFragmentBinding
+
+class CalculatorFragment : Fragment() {
+    private var _binding: CalculatorFragmentBinding? = null
+    private val binding get() = requireNotNull(_binding) { "View was destroyed" }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = CalculatorFragmentBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+
+    @SuppressLint("SetTextI18n")
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        with(binding) {
+            button0.setOnClickListener { textFormula.text = textFormula.text.toString() + "0"}
```

Comment:
у textView есть метод `append`
```suggestion
            button0.setOnClickListener { textFormula.append("0") }
```